ironfish-rust/src/merkle_note.rs

```diff
@@ -199,11 +199,9 @@ impl MerkleNote {
             &self.ephemeral_public_key,
         );
 
-        let mut note_encryption_keys = [0; ENCRYPTED_SHARED_KEY_SIZE];
-        aead::decrypt(
+        let note_encryption_keys = aead::decrypt::<ENCRYPTED_SHARED_KEY_SIZE>(
```

Comment:
Minor nit, up to you. Can also be written as `let note_encryption_keys: [u8; ENCRYPTED_SHARED_KEY_SIZE] = aead::decrypt(...)?`

Reply:
I like it